InterfaceStubGenerator.Core/GeneratedInterfaceStubTemplate.mustache

```diff
@@ -63,7 +63,7 @@ namespace {{Namespace}}
         {
 {{#IsRefitMethod}}    
             var arguments = new object[] { {{ArgumentList}} };
-            var func = methodImpls.GetOrAdd("{{Name}}({{ArgumentListWithTypes}})", _ => requestBuilder.BuildRestResultFuncForMethod("{{Name}}", new Type[] { {{ArgumentTypesList}} }{{#MethodTypeParameterList}}, new Type[] { {{.}} }{{/MethodTypeParameterList}}));
+            var func = methodImpls.GetOrAdd("{{Name}}{{#MethodTypeParameters}}<{{.}}>{{/MethodTypeParameters}}({{ArgumentListWithTypes}})", _ => requestBuilder.BuildRestResultFuncForMethod("{{Name}}", new Type[] { {{ArgumentTypesList}} }{{#MethodTypeParameterList}}, new Type[] { {{.}} }{{/MethodTypeParameterList}}));
```

Comment:
Yeah, this won't work since it will still use the same key for all generic arguments. 😢 

I will take another look soon.